README.md

```diff
@@ -1,5 +1,11 @@
 # Accessible autocomplete
 
+The accessible autocomplete is a component that predicts the rest of a word or string a user is typing.
```

Comment:
I'm not sure about this. It's not just about predicting what they're typing - it's about helping guide users to an answer.

It helps offer suggestions / let users pick from a set of known options. With synonyms it helps find the right answer even when the user uses a different name - that's not the same as predicting what they're typing.

Reply:
It's also about helping standardise responses where people might use different spellings or phrases for the same thing.

Reply:
Thanks @edwardhorsford - your suggestions look good. I'm going to combine them with some other changes I want to make (which it would also be great to get your feedback about).

Reply:
@EoinShaughnessy not remotely meant as suggested text! very little thought went in to it ;)